lua/doom/extras/keybindings/init.lua

```diff
@@ -406,6 +424,17 @@ utils.map(
 --    WhichKey binds     --
 ---]]-----------------[[---
 
+utils.map(
+  "n",
+  "<leader><space>",
```

Comment:
aaah!! what would be another fast way of accessing commandline IYO? How do you do it?
Personally I have been trying many times to reverse ; and : so that commandline is entered
by right pinky press but I never get this to actually work because then ; does not apply
which is sad :(  maybe you know how to do this or something else that is as fast?

Reply:
No idea tbh, I was always manually pressing `:` lol